bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -171,9 +171,11 @@ class PreProcessor {
     concordMetrics::CounterHandle preProcessRequestTimedout;
     concordMetrics::CounterHandle preProcReqSentForFurtherProcessing;
     concordMetrics::CounterHandle preProcPossiblePrimaryFaultDetected;
+    concordMetrics::CounterHandle preProcReqForwardedByNonPrimaryNotIgnored;
```

Comment:
Can we drop this metric, now that we have `metric_total_preexec_requests_executed_` ?